cactusbot/services/beam/parser.py

```diff
@@ -82,6 +82,13 @@ def parse_subscribe(cls, packet):
         return EventPacket("subscribe", packet["user"]["username"])
 
     @classmethod
+    def parse_resubscribe(cls, packet):
```

Comment:
Looks good to me! We should make sure we have a test for this, though. Could you grab a sample packet and add a test to `tests/services/test_beam.py` (after `test_subscribe`)?